os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -0,0 +1,80 @@
+/**************************************************************************************************
+*
+* Copyright 2020 Samsung Electronics All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+* either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*
+***************************************************************************************************/
+
+#include <stdint.h>
+
+#include "flash_api.h"
+#include "device_lock.h"
+
+typedef struct {
+	void (*flash_erase)(uint32_t);
+	int (*flash_read)(uint32_t, uint32_t, uint8_t *);
+	int (*flash_write)(uint32_t, uint32_t, uint8_t *);
+	void (*device_lock)(uint32_t);
+	void (*device_unlock)(uint32_t);
+	int (*get_random_bytes)(void *, uint32_t);
+} ns_func_struc;
```

Comment:
TizenRT ususally using modulename_ops_s, for example i2c_ops_s, spi_ops_s...

Reply:
This is a function pointer structure to hold function pointer and pass in secure region only (static struct), and this structure contains multiple module, it contain, flash, mutex and random generator. If the naming is not suitable please suggest, maybe nsfunc_ops_s?

Reply:
Yes, nsfunc_ops_s looks much better.